.gitattributes

```diff
@@ -5,19 +5,16 @@
 *.jpg binary
 
 # Directories and files we do not want to distribute
-/.build/              export-ignore
-/.ci/                 export-ignore
-/.github/             export-ignore
-/devTools/            export-ignore
-/tests/               export-ignore
-/.editorconfig        export-ignore
-/.gitattributes       export-ignore
-/.gitignore           export-ignore
-/.php_cs.dist         export-ignore
-/.travis.yml          export-ignore
-/appveyor.yml         export-ignore
-/box.json.dist        export-ignore
-/codecov.yml          export-ignore
-/infection.json.dist  export-ignore
-/Makefile             export-ignore
-/phpunit.xml.dist     export-ignore
+/.*                     export-ignore
+/adr/                   export-ignore
+/build/                 export-ignore
+/devTools/              export-ignore
+/tests/                 export-ignore
+/appveyor.yml           export-ignore
+/box.json.dist          export-ignore
+/codecov.yml            export-ignore
+/infection.json.dist    export-ignore
```

Comment:
I would skip those configuration files: no one wants to keep track of them and they are literally just a few KB... It won't impact the resulting artefact in any way

Reply:
No one wants to have them in the project even more.

We can add a test to ensure only allowed paths are in the artefact.

Reply:
>No one wants to have them in the project even more.

It feels to me more an additional burden/pressure putted in the maintainers for absolutely no gain other than a certain sense of purity that you will never achieve.

Removing tests and bigger non-prod directories is acceptable because there is usually not many of them and they don't change often, so having someone updating a `.gitattributes` to get rid of them is ok. But going further and trying to trim the git artefact from absolutely all unnecessary files is an unnecessary strain for absolutely no value (you will gain maybe 100KB max from it which is nothing)

You could add a test, but it doesn't remove the problem: it means further friction when contributing/maintaining because you may need to update that test, it means having an additional test/code for it which is to be maintained as well... So again, for 100KB it's just not worth it IMO